### DISCUSSION TITLE
sync transform

### DISCUSSION THREAD

**Discussion by o-az** [parcel-bundler/lightningcss#720] [Q&A] [answered]
Is there a way to accomplish `transform` synchronously? I see `bundle` has `sync` option. would love to have one for `transform`

**Comment by mischnic** [comment, marked as answer]
On https://lightningcss.dev/docs.html, the first example is indeed a synchronous `transform`?

```js
import { transform } from 'lightningcss';

let { code, map } = transform({
  filename: 'style.css',
  code: Buffer.from('.foo { color: red }'),
  minify: true,
  sourceMap: true
});
```
